src/Microsoft.NetCore.Analyzers/Core/Security/SourceTriggeredTaintedDataAnalyzerBase.cs

```diff
@@ -91,97 +92,69 @@ public override void Initialize(AnalysisContext context)
                                 operationAnalysisContext =>
                                 {
                                     IInvocationOperation invocationOperation = (IInvocationOperation)operationAnalysisContext.Operation;
-                                    PooledHashSet<IsInvocationTaintedWithPointsToAnalysis> evaluateWithPointsToAnalysis = null;
-                                    PooledHashSet<IsInvocationTaintedWithValueContentAnalysis> evaluateWithValueContentAnalysis = null;
+                                    IOperation rootOperation = operationAnalysisContext.Operation.GetRoot();
+                                    PooledDictionary<PointsToCheck, ImmutableHashSet<string>> evaluateWithPointsToAnalysis = null;
+                                    PooledDictionary<ValueContentCheck, ImmutableHashSet<string>> evaluateWithValueContentAnalysis = null;
+                                    PointsToAnalysisResult pointsToAnalysisResult = null;
+                                    ValueContentAnalysisResult valueContentAnalysisResult = null;
+                                    if (rootOperation.TryGetEnclosingControlFlowGraph(out ControlFlowGraph cfg))
+                                    {
+                                        pointsToAnalysisResult = PointsToAnalysis.TryGetOrComputeResult(
```

Comment:
This is force computing interprocedural points to analysis for every method. We should not be so aggressive in kicking off DFA.

Reply:
Good catch.  I'll try to make this smarter today.